src/libAtomVM/bif.c

```diff
@@ -42,9 +42,12 @@
     ctx->x[1] = (error_type_atom);   \
     return term_invalid_term();
```

Comment:
What about changing `RAISE_ERROR` to `RAISE_ERROR_BIF`?
e.g.:
```c
#define RAISE_ERROR_BIF(fail_label, error_type_atom) \
if (fail_label) { \
    ctx->x[0] = ERROR_ATOM;          \
    ctx->x[1] = (error_type_atom);   \
} \
    return term_invalid_term();
```

Reply:
Thank you for the suggestion